src/main/resources/hazelcast-config-schema.json

```diff
@@ -1373,22 +1373,129 @@
             "$ref": "#/definitions/AsyncBackupCount"
           },
           "item-listeners": {
-          "type": "array",
-            "items":  {
+            "type": "array",
+            "items": {
               "$ref": "#/definitions/ItemListener"
             }
           },
           "split-brain-protection-ref": {
             "type": "string",
-            "description":  "Adds the Split Brain Protection for this data-structure. You should set this value as a \"split-brain-protection\"'s name."
+            "description": "Adds the Split Brain Protection for this data-structure. You should set this value as a \"split-brain-protection\"'s name."
           },
           "merge-policy": {
             "$ref": "#/definitions/MergePolicy"
           }
         }
       }
+    },
+    "NativeMemory": {
+      "allOf": [
+        {
+          "description": "This feature is available only in Hazelcast Enterprise HD. Configuration element's name is \"native-memory\".",
```

Comment:
I think we can remove the "Configuration element's name is \"native-memory\"." sentence, it doesn't convey useful information.

Reply:
removed